config-model/pom.xml

```diff
@@ -308,6 +308,14 @@
       <artifactId>tensorflow</artifactId>
       <version>1.6.0</version>
     </dependency>
+    <dependency>
```

Comment:
You shouldn't need to add these dependencies to this module?

Reply:
I have tried to do this in kind of "compatible" way, but given small mileage on vespa I do not really know all the concerns.
I could do the OpenNlpDetector and OpenNlpTokenizer but if I create a new linguistics class then it will not be used anywhere which makes the whole change kind of pointless to me.
Do you recommend to fallback to the KSTem in the  OpenNlpTokenizer for english or use just OpenNlp provided stuff for stemming?

Reply:
It's not pointless, this is a great contribution but we cannot change the default stemmer in a minor release as it would cause potential recall issues as the index would need to be re-processed. With the suggested approach by @bratseth users can enable this new great functionality in Vespa 6 by configuration while we considering changing the defaults for Vespa 7.

Reply:
Also, we'll add this to the documentation and create a blog post on it.